### PR TITLE
Add store xml samples

### PR DESCRIPTION
**Pull request by ramsai1729**
This PR adds two store xml samples 

- Creating clustered store using XML configuration
- Exporting programmatic configuration to XML 

Please review.

### CODE REVIEW

File: store-example06-config-export/start-server.bat [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+@REM   Unless required by applicable law or agreed to in writing, software
+@REM   distributed under the License is distributed on an "AS IS" BASIS,
+@REM   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+@REM   See the License for the specific language governing permissions and
+@REM   limitations under the License.
+
+@echo off
+setlocal
+
+set WD=%~d0%~p0
```
Comment (cljohnso) [inline store-example06-config-export/start-server.bat]: This line does not mirror the operation in the corresponding shell script.  Use
```
pushd "%~dp0"
set WD=%CD%
popd
```
instead.

File: ehcache-example03-xml/start-client.bat [hunk 2]
```diff
@@ -42,7 +42,7 @@ if not defined JAVA_HOME (
 set JAVA="%JAVA_HOME%\bin\java.exe"
 set JAVAC="%JAVA_HOME%\bin\javac.exe"
 
-set TC_CP="%WD%\src"
+set TC_CP="%WD%\src;%WD%\resources"
```
Comment (anthonydahanne) [inline ehcache-example03-xml/start-client.bat:45]: ah, I removed it doing my find / replace - thanks

**Review by cljohnso — CHANGES_REQUESTED** [review]

File: store-example06-config-export/start-server.bat (hunk 1 above)
Comment (cljohnso) [reply store-example06-config-export/start-server.bat]: The rest of the changes look good.

**Comment by ramsai1729** [comment]
@cljohnso fixed now.

**Review by cljohnso — APPROVED** [review]

**Comment by anthonydahanne** [comment]
works fine on mac os

**Comment by anthonydahanne** [comment]
and both examples also work fine on windows

**Pull request merged**